Bixal/move.mil
# SME sorting and section title configuration

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Fixes Personal Property title
- sorts by Document ID (descending)
- Order documents on SME page Alphabetically

## Testing

To verify the changes proposed in this pull request…

1. Import configuration
1. Verify SME looks as expected.
